app/controllers/api/v2/scc_accounts_controller.rb

```diff
@@ -34,7 +34,7 @@ def show; end
           param :password, String, :required => true, :desc => N_('Password of scc_account')
           param :base_url, String, :required => false, :desc => N_('URL of SUSE for scc_account')
           param :interval, ['never', 'daily', 'weekly', 'monthy'], :desc => N_('Interval for syncing scc_account')
-          param :sync_date, String, :desc => N_('Last Sync time of scc_account')
+          param :sync_date, String, :desc => N_('Sync date')
```

Comment:
```suggestion
          param :sync_date, String, :desc => N_('Date and time relative to which the sync interval is run')
```

Reply:
I like that. Maybe, we should add a help field on the `scc_account` field with the same information. Although, of course, it is also written in the documentation.